.env

```diff
@@ -55,6 +54,11 @@ PLACE_USERNAME="Place Support (localhost:8443)"
 
 PLACE_SERVER_SECRET=development
 
+# Staff API variables
+
+POSTGRES_USER=placeos
+POSTGRES_PASSWORD=development
```

Comment:
Tagging this for neaten up as part of #8

Reply:
yep noted thanks

Reply:
All good. There's static secrets in use elsewhere too - this was just so this this one pops up against that issue when this is solved generally. No action needed as part of this PR.